runtime/runtime/Cargo.toml

```diff
@@ -19,14 +23,14 @@ ethereum-types = "0.11.0"
 
 borsh = "0.8.1"
 
-near-crypto = { path = "../../core/crypto" }
-near-primitives = { path = "../../core/primitives" }
-near-store = { path = "../../core/store" }
-near-metrics = { path = "../../core/metrics" }
-near-runtime-utils = { path = "../../runtime/near-runtime-utils" }
-near-vm-logic = { path = "../../runtime/near-vm-logic" }
-near-vm-runner = { path = "../../runtime/near-vm-runner" }
-near-vm-errors = { path = "../../runtime/near-vm-errors" }
+near-crypto = { path = "../../core/crypto", version = "0.1.0" }
+near-primitives = { path = "../../core/primitives", version = "0.1.0"}
+near-store = { path = "../../core/store", version = "2.2.0" }
+near-metrics = { path = "../../core/metrics", version = "0.1.0" }
+near-runtime-utils = { path = "../../runtime/near-runtime-utils" , version = "3.0.0" }
+near-vm-logic = { path = "../../runtime/near-vm-logic", version = "3.0.0" }
+near-vm-runner = { path = "../../runtime/near-vm-runner", version = "3.0.0" }
+near-vm-errors = { path = "../../runtime/near-vm-errors", version = "3.0.0" }
 near-evm-runner = { path = "../../runtime/near-evm-runner", optional = true }
```

Comment:
It seems that it needs to be updated with the version as well.

Reply:
I was going to ask about this, but I was hoping to publish the runtime _without_ `near-evm-runner` because it has github deps.

@frol @matklad is this possible?   If not, then I'll have to publish the two github deps it uses.

Reply:
I'd just comment it out before publishing )

Reply:
LPT! Thanks!